int8 direction        #  1 = forwards; -1 = backwards
float32 speed         # m/s    (must be >=Â 0)
---
bool success